### PR TITLE
feat(kms): add samples for new rng and hmac APIs

### PR DESCRIPTION
**Pull request by sethvargo**
This adds samples for new KMS APIs.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,7 +3,7 @@ module github.com/GoogleCloudPlatform/golang-samples
 go 1.11
 
 require (
-	cloud.google.com/go v0.90.0
```
Comment (sethvargo) [inline go.mod:6]: The mod changes are from updating the kms and kmspb packages + `go mod tidy`

**Comment by sethvargo** [comment]
I'm not entirely sure what CI is complaining about with respect to the go.mod. Any ideas

**Comment by codyoss** [comment]
@sethvargo Hope you don't mind a pushed a commit to see if it is all just tidy related. Because this is a multi-module project where some of the sub-modules depend on the root module with replace statements -- if you modify the root go.mod you have to tidy any with a replace statement. I have an bash function to help me with this: 

```
tidyall() {
    go mod tidy
    wd="$PWD"
    for i in $(find $wd -name go.mod); do
        pushd $(dirname $i)
        go mod tidy
        popd
    done
    cd $wd
}
```

**Review by codyoss — APPROVED** [review]
LGTM

**Comment by tbpg** [comment]
One liner (mentioned in the error message):
```
find . -name go.mod -execdir go mod tidy \;
```

**Comment by sethvargo** [comment]
Sweet thank you both!

**Comment by sethvargo** [comment]
Hmm - is the go111 failure related?

**Comment by codyoss** [comment]
Let me try to push one more commit to see if we can get this working. It's looks like with the changes in the go.mod a transitive dep now requires Go 1.13, so 1.11 will fail.

**Comment by sethvargo** [comment]
Thank you @codyoss - let me know how I can help.

**Comment by codyoss** [comment]
Forgot to come back to this yesterday sorry. It was the use of `%w` in go1.11 test that was causing the test failures. Moved usage to `%v`.

**Comment by sethvargo** [comment]
Oh my b. Thank you for catching that and thank you for merging!

**Pull request merged**